java_tests/src/test/java/org/infinispan/client/hotrod/DotNetServerRestartTest.java

```diff
@@ -0,0 +1,47 @@
+package org.infinispan.client.hotrod;
+import org.infinispan.util.logging.Log;
+import org.infinispan.client.hotrod.test.HotRodClientTestingUtil;
+import org.infinispan.client.hotrod.ServerRestartTest;
+import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
+import org.infinispan.server.hotrod.HotRodServer;
+import org.infinispan.server.hotrod.configuration.HotRodServerConfigurationBuilder;
+import org.infinispan.util.logging.LogFactory;
+public class DotNetServerRestartTest extends ServerRestartTest
```

Comment:
I believe these two classes in this commit need better formatting (empty lines missing, using tabs instead of spaces).